src/test/finalization/state_repository_tests.cpp

```diff
@@ -366,7 +366,7 @@ BOOST_AUTO_TEST_CASE(recovering) {
     restored_repo->RestoreFromDisk(proc.get());
     BOOST_CHECK(fixture.m_block_db.blocks.empty());
     LOCK(restored_repo->GetLock());
-    BOOST_CHECK(restored_repo->Find(*fixture.m_chain.AtHeight(5)) != nullptr);
+    BOOST_CHECK(restored_repo->Find(*fixture.m_chain.AtHeight(4)) != nullptr);
```

Comment:
The reason for the change is that the finalized point was moved.

Reply:
This PR ships a zillion one-line changes of numbers. Looking deeper in this particular file I don't see a reason for any change here.

Reply:
Yeah, I checked that more precisely. The problem is that we trim repository at the last step of finalization recovering (`RepositoryImpl::CheckAndRecover`).